src/ui/common/src/components/integrations/dialogs/dialog.tsx

```diff
@@ -146,7 +146,9 @@ const IntegrationDialog: React.FC<Props> = ({
     Object.values(state.integrationsReducer.integrations)
   );
 
-  const numWorkflows = new Set(operators.map((x) => x.workflow_id)).size;
+  const numWorkflows = operators
+    ? new Set(operators.map((x) => x.workflow_id)).size
+    : 0;
```

Comment:
@agiron123 this is a small change to the dialog component that stopped email resources from crashing on any edits or deletes, which was a pain when developing. I don't think it should mess up what you've been doing.